cli_reference/basic_cli_operations.adoc

```diff
@@ -658,6 +658,13 @@ Launch a command shell to debug a running application.
 $ oc debug [Options]
 ----
 
+For example, for the `oc debug <pod>|<object_type/object_name> [options] [-- command]` command:
+
+* `debug` is the action performed by the command.
+* `<object_type>` is the service or abbreviated svc the action is performed on.
+* `<object_name>` is the specified <object_type>.
+* `[options]` passes parameters to the program.
```

Comment:
**\`<object_type>\` is the service or abbreviated svc
\`<object_name>\` is the specified <object_type>**
Hmm, the statement is not understandable.
To be clearer and simpler, now suggest the update just replace the line "oc debug [Options]" with "oc debug \<pod\> [options] [-- command]". I think this is OK enough